docs/background-checks.md

```diff
@@ -27,6 +27,20 @@ console.log(check);
 
 With background checks enabled, Medicus will automatically run health checks on a consistent interval, but you can still manually trigger a check whenever necessary using `performCheck()`.
 
+## Immediate Execution with `eagerBackgroundCheck`
+
+When you enable background checks, you can control whether an initial health check runs immediately upon initialization or if Medicus should wait for the first interval to elapse. This behavior is controlled by the eagerBackgroundCheck option, which defaults to true.
+
```

Comment:
```suggestion

This is especially useful to avoid unintentionally flagging containers with downtime when during deployment of new versions, since it would restart the timeout once the container starts back up again, delaying the first health report.

```